src/main/java/sparksoniq/semantics/DynamicContext.java

```diff
@@ -103,14 +103,14 @@ public void setBindingsFromTuple(FlworTuple tuple, IteratorMetadata metadata) {
         return _rddVariableValues.keySet();
     }
 
-    public Set<String> getDFKeys() {
-        return _dfVariableValues.keySet();
+    public Set<String> getDataFrameKeys() {
```

Comment:
getDataFrameVariableNames (key is a low-level detail).

Reply:
Great suggestion